src/components/NcRichContenteditable/NcRichContenteditable.vue

```diff
@@ -47,16 +47,28 @@ This component displays contenteditable div with automated `@` [at] autocompleti
 			:user-data="userData"
 			placeholder="Try mentioning user @Test01 or inserting emoji :smile"
 			@submit="onSubmit" />
-		<br>
-		<br>
+
+		<h5>Output - raw</h5>
 		{{ JSON.stringify(message) }}
+
+		<h5>Output - preformatted</h5>
+		<p class="pre-line">
+			{{ message }}
+		</p>
```

Comment:
To remove two addition blank lines without `-20px` margin
```suggestion
		<p class="pre-line">{{ message }}</p>
```